_includes/rest/objects.md

```diff
@@ -501,7 +501,7 @@ print(result)
 
 ## Batch Operations
 
-To reduce the amount of time spent on network round trips, you can create, update, or delete up to 50 objects in one call, using the batch endpoint.
+To reduce the amount of time spent on network round trips, you can create, update, or delete using the batch endpoint.
```

Comment:
Should we add this?

```suggestion
To reduce the amount of time spent on network round trips, you can create, update, or delete using the batch endpoint. The batch size can be customized, the default batch size is 20.
```

Reply:
Yes, that makes sense. I'll add that.

Reply:
Oh you could just have commited the suggestion with the button above :-)